airbyte-webapp/src/views/layout/SideBar/SideBar.module.scss

```diff
@@ -2,6 +2,8 @@
 @use "../../../scss/variables";
 
 .menuItem {
+  background-color: transparent;
```

Comment:
:art: If we apply this to the `button` element, `appearance: none` might be worth adding, which tells the browser to not apply the `button` specific default user agent styles.